svf/include/SVFIR/SVFVariables.h

```diff
@@ -689,6 +685,676 @@ class FunObjVar : public FIObjVar
     virtual const std::string toString() const;
 };
 
+
+/*
+ *  Constant objects, including ConstantValVar inherited from ValVar,
+ *  and ConstantObjVar inherited from FIObjVar
+ */
+class ConstantValVar: public ValVar
+{
+    friend class SVFIRWriter;
+    friend class SVFIRReader;
+
+public:
+    ///  Methods for support type inquiry through isa, cast, and dyn_cast:
+    //@{
+    static inline bool classof(const FunValVar*)
+    {
+        return true;
+    }
+    static inline bool classof(const ValVar* node)
+    {
+        return node->getNodeKind() == ConstantValNode;
```

Comment:
this classof should be a range, you can refer to https://github.com/SVF-tools/SVF/blob/master/svf/include/SVFIR/SVFVariables.h#L283